.gitmodules

```diff
@@ -369,3 +366,10 @@
 [submodule "contrib/numactl"]
 	path = contrib/numactl
 	url = https://github.com/ClickHouse/numactl.git
+[submodule "contrib/ggml"]
+	path = contrib/ggml
+	url = https://github.com/ggerganov/ggml
+[submodule "contrib/replxx"]
+	path = contrib/replxx
+	url = https://github.com/iyubondyrev/replxx
```

Comment:
We should be using ClickHouse fork here so please submit these changes to ClickHouse fork first. :)

Reply:
> I will be updating this PR description and the code because it is still ongoing.

Ahh I didn't read this part of your pull request... You are still developing this feature 🤦  I suppose you will make the pull request later when it's ready, ... I will leave reviewing the actual implementation to our core engineers at ClickHouse.

Reply:
> We should be using ClickHouse fork here so please submit these changes to ClickHouse fork first. :)

[Done](https://github.com/ClickHouse/replxx/pull/28)